tests/param_table_test.py

```diff
@@ -6,22 +6,26 @@
 
 
 def test_raises_with_zero_columns() -> None:
-    with pytest.raises(ValueError):
+    with pytest.raises(
+        ValueError, match="Cannot create param table with zero columns and no index"
+    ):
         sl.ParamTable(row_dim=int, columns={})
 
 
 def test_raises_with_inconsistent_column_sizes() -> None:
-    with pytest.raises(ValueError):
+    with pytest.raises(
+        ValueError, match="Columns in param table must all have same size"
```

Comment:
Not sure if we should put the exact error message here. This means that the test needs to be updated when the message changes. (Here, in particular, there is a grammatical error...)

Reply:
I think we should definitely put them in. We use generic errors like `ValueError` and with a catch-all test we can't really be sure if the error is being raised at the correct place.

Reply:
I mean that I wouldn't put the exact error message but instead a pattern that checks for expected terms.